learn/advanced/known_limitations.md

```diff
@@ -88,3 +88,13 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 **Limitation:** Meilisearch can only exactly represent integers between -2⁵³ and 2⁵³.
 
 **Explanation:** Meilisearch stores numeric values as double-precision floating-point numbers. This allows for greater precision and increases the range of magnitudes that Meilisearch can represent, but leads to inaccuracies in [values beyond certain thresholds](https://en.wikipedia.org/wiki/Double-precision_floating-point_format#Precision_limitations_on_integer_values).
+
+## Maximum number of results per search
+
+**Limitation:** Meilisearch returns 1000 documents per search.
+
+**Explanation:** This non-customizable limit ensures the database is protected from scraping. This limit only applies to the [search route](/reference/api/search.md). If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.
+
+:::note
+This limit is different from the [`limit` search parameter](/reference/api/search.md#limit). Whereas `limit` restricts the amount of results sent to the front-end, this means only 1000 results exist.
```

Comment:
I am not sure to understand the "this means only 1000 results exist" part here.

Reply:
From the user's perspective, Meilisearch only has 1000 search results. Is this technically inaccurate?

Reply:
Yes, it is correct but I prefer the way you just explained it to me in your comment :)